lib/internal/bootstrap/node.js

```diff
@@ -411,6 +412,24 @@
     });
   }
 
+  function setupGlobalEncoding() {
+    const { TextEncoder, TextDecoder } = NativeModule.require('util');
+    Object.defineProperties(global, {
+      TextEncoder: {
+        value: TextEncoder,
+        writable: true,
+        configurable: true,
+        enumerable: true
```

Comment:
Per [Web IDL](https://heycam.github.io/webidl/#es-interfaces) the properties should not be enumerable.

Reply:
ah right... I knew something was off about that. Thx!